src/helpers/genCommaList.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Given an array of strings, create a single comma-separated string that includes
+ * 'and' as well as an oxford comma.
+ * Ex: ['apples'] => 'apples'
+ * Ex: ['apples', 'bananas'] => 'apples and bananas'
+ * Ex: ['apples', 'bananas', 'grapes'] => 'apples, bananas, and grapes'
+ * @author Austen Money
+ * @param list an array of elements to be made into a single comma-separated string.
+ * @returns a comma-separated string.
+ */
+const genCommaList = (list: string[]): string => {
+  const { length } = list;
+
+  return (
+    length < 2
```

Comment:
To keep things standard with other projects, we don't usually destructure `length`. Could you put this as `list.length < 2`?